packages/o-spreadsheet-engine/src/types/misc.ts

```diff
@@ -119,6 +119,7 @@ export interface Style {
   fillColor?: Color;
   textColor?: Color;
   fontSize?: number; // in pt, not in px!
+  rotation?: number;
```

Comment:
in degrees ? radians ? also explain the coordinate system (reversed from usual math)